db/db_test.cc

```diff
@@ -247,17 +247,19 @@ TEST_F(DBTest, SkipDelay) {
       wo.sync = sync;
       wo.disableWAL = disableWAL;
       wo.no_slowdown = true;
-      dbfull()->Put(wo, "foo", "bar");
+      // Large enough to exceed allowance for one time interval
+      std::string large_value(1024, 'x');
+      dbfull()->Put(wo, "foo", large_value).PermitUncheckedError();
```

Comment:
Should we check the result of `Put()`?

Reply:
My understanding is that it would be nice if the first large write triggered a delay, but for now it requires a second write.

Reply:
I can check for that and make the comment more explicit what might be OK or GOOD progress in the case of failure.